src/main/resources/i18n/displayStrings.properties

```diff
@@ -1501,7 +1501,7 @@ systemTray.tooltip=Bisq: The decentralized exchange network
 ####################################################################
 
 guiUtil.miningFeeInfo=Please be sure that the mining fee used at your external wallet is \
-at least {0} Satoshi/Byte. Otherwise the trade transactions cannot be confirmed and a trade would end up in a dispute.
+at least {0} Satoshis/byte. Otherwise the trade transactions cannot be confirmed and a trade would end up in a dispute.
```

Comment:
For the same reasons that the word "bitcoin" is spelled with a lower case 'b' when referring to bitcoin as a unit of currency, so too should "satoshi" be spelled with a lowercase 's'. For reference, see https://en.bitcoin.it/wiki/Satoshi_(unit).

Reply:
And with regard to pluralizing as "satoshis", this appears to be fine, and I agree it reads better in this context. See https://en.bitcoin.it/wiki/Satoshi_(unit)#Plural.

We may also want to type out the phrase "satoshis per byte" instead of "satoshis/byte".